extensions/wikia/CreateNewWiki/tasks/SetCustomSettings.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Wikia\CreateNewWiki\Tasks;
+
+use User;
+use WikiFactory;
+use Wikia\Logger\Loggable;
+
+class SetCustomSettings implements Task {
+	use Loggable;
+
+	private $taskContext;
+
+	public function __construct( TaskContext $taskContext ) {
+		$this->taskContext = $taskContext;
+	}
+
+	public function prepare() {
```

Comment:
Maybe we could just create a base class with these three prepare/check/run methods returning success, so we don't need to create them in each task if they're empty?